config/webpack/environment.js

```diff
@@ -1,3 +1,11 @@
 const { environment } = require('@rails/webpacker')
 
+const webpack = require('webpack')
+environment.plugins.append('Provide',
+  new webpack.ProvidePlugin({
+    $: 'jquery',
```

Comment:
what is the purpose of this change? if you need to require poper.js and jquery

Reply:
The reason is same I was following bootstrap using webpacker link in which this changes was required.Thats why I was confused while require('bootstrap') and think all these are required for bootstrap.But now I have understand.